libs/sdk-backend-tiger/src/backend/workspace/settings/index.ts

```diff
@@ -34,6 +36,67 @@ export class TigerWorkspaceSettings implements IWorkspaceSettingsService {
     public getSettingsForCurrentUser(): Promise<IUserWorkspaceSettings> {
         return getSettingsForCurrentUser(this.authCall, this.workspace);
     }
+
+    public async setSetting(id: string, content: any): Promise<void> {
+        // Currently it is necessary to check existence of required setting
+        // since PUT does not support creation of non-existing setting.
+        // It can be simplified to Update method once NAS-4291 is implemented
+        try {
+            await this.getSetting(id);
+            await this.updateSetting(id, content);
+        } catch (error: any) {
```

Comment:
IMHO this error type is known. Same for user settings one.

Reply:
good point